src/peft/utils/other.py

```diff
@@ -550,6 +556,41 @@ def set_adapter(self, adapter_names: Union[str, list[str]]):
         self.modules_to_save[adapter_name].requires_grad_(True)
         self._active_adapter = adapter_name
 
+    def delete_adapter(self, adapter_name: str, new_active_adapters: Optional[list[str]]) -> None:
+        """
+        Delete the adapter if present.
+
+        This method will also set a new active adapter if the deleted adapter was the active adapter. It is important
+        that the new adapter is chosen in a deterministic way, so that the same adapter is chosen on all layers.
```

Comment:
```suggestion
        This method will also set a new active adapter if the deleted adapter was the active adapter. It is important
        that the new adapter is chosen by the caller in a deterministic way, so that the same adapter is chosen on all layers.
```

This is something that the caller has to watch out for, as I understand it. Or am I wrong?